src/main/java/FigureInterface.java

```diff
@@ -0,0 +1,5 @@
+public interface FigureInterface {
+    abstract void draw();
+
+    public abstract double areaValue();
```

Comment:
What about method naming? And we can omit access modifiers in interfaces, they are public by default